src/core/Board.module.scss

```diff
@@ -1,7 +1,16 @@
 .board {
-  display: grid;
-  gap: 30px;
-  grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
-  margin: auto;
-  width: calc((155px * 8))
+  padding: 100px;
+
+  &__cards--container {
+    justify-content: center;
+    display: grid;
+    gap: 30px;
+    grid-template-columns: repeat(auto-fill, minmax(120px, 1fr));
```

Comment:
Grids are great, but do be careful because they are not supported in every browser. On some projects, you will not be able to use grids.

Reply:
Around the time I left on mat leave the X team had _just_ decided that browser support was enough to start using grids in production for marketing sites (with graceful degradation) but I do think I might update this to flex a little down the line; as it is if I have a few cards on a large screen they don't centre properly because autofill creates more (empty) grid columns than I need. But I'm going to save that for a little later.